test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NetworkCallCountTest.cs

```diff
@@ -1199,7 +1199,7 @@ public void NetworkCallCount_RestoreSolutionMultipleSourcesTwoV3()
                     "-Verbosity",
                     "detailed",
                     "-DisableParallelProcessing",
-                    "MSBuildVersion",
+                    "-MSBuildVersion",
```

Comment:
Why this changed?

Reply:
It seems that, in the past, the test only worked in that specific version of MSBuild. By removing that argument, the test worked well.

Reply:
Looking into the test, I don't think we should be using a specific msbuild version. 

NuGet.exe should roll forward seemlessly. 

I'd consider removing it, but you don't have to do it in this PR.